CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+Version 11.0
```

Comment:
I think here it is actually not breaking because we have used `JobStatus` all along, so I would suggest a minor version update.

Reply:
Updated version number.